prov/verbs/src/fi_verbs.c

```diff
@@ -319,10 +319,16 @@ fi_ibv_send(struct fi_ibv_msg_ep *ep, struct ibv_send_wr *wr, void *context)
 
 	if (ep->scq) {
 		if (wr->send_flags & IBV_SEND_SIGNALED) {
+			fastlock_init(&ep->wre_lock);
```

Comment:
fastlock_acquire

Reply:
@plebioda Thanks. Looks strange why it doesn't fail. My tests is ok